### DISCUSSION TITLE
Fluent-Metro On Windows 8.1!

### DISCUSSION THREAD

**Discussion by angelicwine** [bonzibudd/Fluent-Metro#177] [General]
I have used Fluent-Metro many times before upgrading to Windows 11 on my Windows 10 machine. Then today, I was messing around with a Windows 8.1 VM and saw that it said: "incompatible with Windows 7 and 8". I decided to try it and even the tiles work. Here's some [image
![wow](https://user-images.githubusercontent.com/84406551/184973239-a87cf41d-aa65-4919-b9de-becca01fe3a6.png)
.

**Comment by bonzibudd** [comment]
Yes, it is mostly functional but I can’t recommend use on 8 and 8.1 for general users because several layouts and visual features are broken below Win10. However, all of the default settings do work on 8.1 from what I remember.
